.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:centos7
```

Comment:
I would say that if you plan on submitting a PR back to the php maintainers, you should remove the `.devcontainer` changes here

Reply:
No plans to submit our changes upstream but I also intend to remove the devcontainer config before we begin creating a new RPM for deployment.

The devcontainer config was included as a convenience for review and QA.